Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:7.8-alpine
```

Comment:
It is just a matter of operating system base image, the alpine one is much smaller:

Alpine version:

```
analytics-reporter                                      latest                                     1b31d8a1afab        8 seconds ago       117 MB
```

Ubuntu version:

```
analytics-reporter                                      latest                                     39d0e3929a42        34 minutes ago      723 MB
```

Ubuntu adds 600MB of operating system things, to me it's better to stick with alpine in this specific case.